onnx/onnx-operators_pb.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Facebook Inc. and Microsoft Corporation.
+// Licensed under the MIT license.
+
+#ifndef ONNX_ONNX_OPERATOR_PB_H
+#define ONNX_ONNX_OPERATOR_PB_H
+
```

Comment:
This file also needs [these boilerplates](https://github.com/onnx/onnx/blob/master/onnx/onnx_pb.h#L7-L59)